apps/api-documenter/package.json

```diff
@@ -11,8 +11,8 @@
   "license": "MIT",
   "scripts": {
     "build": "heft build --clean",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
```

Comment:
This command is saying "Clean the stuff related to tests and run tests" right?

Reply:
Yes correct, it doesn't mean "clean all the things", just the things relevant to the running phases.